os/other-settings.md

```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
+
+This example will load `dummy` module with `numdummies=5` option:
+
+```yaml
+#cloud-config
+
+write_files:
+  - path: /etc/modprobe.d/dummy.conf
+    content: options dummy numdummies=5
+  - path: /etc/modules-load.d/dummy.conf
+    content: dummy
+
+coreos:
+  units:
+    - name: systemd-modules-load.service
+      command: restart
+```
+
+As a result you will get five dummy network interfaces.
```

Comment:
After this cloud-config is processed, five `dummy` interfaces are added to the network stack.